websites/B/Baseball-Reference/presence.ts

```diff
@@ -0,0 +1,92 @@
+const presence = new Presence({
+		clientId: "1049610101725605899",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	{ href, pathname, hostname } = document.location;
+
+let team: string = null,
+	image = document.images.item(1).src;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		startTimestamp: browsingTimestamp,
+	};
+	presence.setActivity(presenceData);
+	if (hostname === "baseball-reference.com") {
+		if (pathname === "/") presenceData.details = "Viewing Home Page";
+	} else if (pathname.includes("/players")) {
+		presenceData.details = "Viewing a Player";
+		presenceData.state = `${
+			document.querySelector("div#meta div h1 span").textContent
+		} | ${document
+			.querySelector("div#meta div p strong")
+			.nextSibling.textContent.replace(/\s/g, " ")
+			.trim()}`;
+		if (!presence.getActivity().largeImageKey)
+			presenceData.largeImageKey = image;
+
+		presenceData.buttons = [{ label: "View Player", url: href }];
+
+		if (
+			document.querySelector("#meta > div:nth-child(2) > p:nth-child(5) > a")
+		) {
+			team = document.querySelector(
+				"#meta > div:nth-child(2) > p:nth-child(5) > a"
+			).textContent;
+			presenceData.state = `${presenceData.state} For The ${team}`;
```

Comment:
I think an unexpected output will be caused by doing this, can you prove to me it won't through a screenshot?